### PR TITLE
Diagnostics html

### PR DESCRIPTION
**Pull request by stephpenn1**
- Added daily averages per collar in **licorplot.R**
- Updated URL to html

### CODE REVIEW

File: R_scripts/licorplot.R [hunk 1]
```diff
@@ -13,41 +13,51 @@ library(readr)
 source("licordat.R")
 licorDat <- read_dir("../licor_data/")
 collarDat <- read_csv("../design/cores_collars.csv")
+plots <- read_csv("../design/plots.csv")
```
Comment (stephpenn1) [inline R_scripts/licorplot.R]: Is there a huge difference between _read_csv_ and _read.csv_?

Comment (bpbond) [reply R_scripts/licorplot.R]: `read.csv` is the R default, and produces a data frame. It's robust but not fast.

`read_csv` is provided by the `readr` package. It's a little more fragile in some ways, but much faster, produces a tibble, and automatically handles things like dates and times.

**Comment by bpbond** [comment]
I resolved conflicts. Also changed the top of `diagnostics.Rmd` to

```
# Diagnostics as of `r date()`
```

which looks better.

File: R_scripts/diagnostics.Rmd [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+plots <- read.csv("../design/plots.csv")
+knitr::kable(plots) 
+```
+
+|Location |Experiment           |No. of Collars |
+|:-------:|:-------------------:|:-------------:|
+|SERC     |Control              |56             |
+|SERC     |Salinity transplant  |36             |
+|SERC     |Elevation transplant |32             |
+
```
[the remark this answers is not in the document]
Comment (bpbond) [reply R_scripts/diagnostics.Rmd]: Agreed!

**Pull request merged**